compiler/types.nim

```diff
@@ -1532,7 +1532,7 @@ proc getProcConvMismatch*(c: ConfigRef, f, a: PType, rel: var TTypeRelation): se
        # but it's a pragma mismatch reason which is why it's here
        result.incl pcmLockDifference
 
-proc getProcConvMismatch*(c: ConfigRef, f, a: PType): set[ProcConvMismatch] =
+proc getProcConvMismatch*(c: ConfigRef, f, a: PType): set[ProcConvMismatch] {.inline.}=
```

Comment:
oh i didn't mean add inline, I meant inline the code in the sense that it's only used once so move the body to where it's being used